src/client/components/app-core/app-core.stories.css

```diff
@@ -0,0 +1,39 @@
+.story {
+  padding: 2rem;
+}
+
+.title,
+.subtitle {
+  margin-bottom: 2rem;
+  padding-bottom: 1rem;
+  border-bottom: 1px solid #c5c5c5;
+}
+
+.subtitle {
```

Comment:
in general we should avoid using `margin-top` to avoid [collapsing margins](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing).